src/grt/grt-wave_opt_file-parse-debug.ads

```diff
@@ -33,6 +33,6 @@ private package Grt.Wave_Opt_File.Parse.Debug is
    procedure Dump_Tree;
 
    -- Dump recursively an element of the tree.  Called by Dump_Tree
-   procedure Dump_Sub_Tree (Cursor : Elem_Acc; Level : Positive);
+   procedure Dump_Sub_Tree (Cursor : Elem_Acc);
```

Comment:
Fixed


Le 27. 07. 16 à 05:19, tgingold a écrit :
>
> In src/grt/grt-wave_opt_file-parse-debug.ads 
> <https://github.com/tgingold/ghdl/pull/124#discussion_r72375157>:
>
> > @@ -33,6 +33,6 @@ private package Grt.Wave_Opt_File.Parse.Debug is
> >     procedure Dump_Tree;
> >
> >     -- Dump recursively an element of the tree.  Called by Dump_Tree
> > -   procedure Dump_Sub_Tree (Cursor : Elem_Acc; Level : Positive);
> > +   procedure Dump_Sub_Tree (Cursor : Elem_Acc);
>
> If Dump_Sub_Tree is not called from another package, no need to 
> declare it in the package spec.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/tgingold/ghdl/pull/124/files/77c854621cce3da7afdd08837584aea3fe27c1ca#r72375157>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/ATNePt88Jw1YG18AzPTqiP6G69p_rqDzks5qZs4lgaJpZM4JVgTD>.
>